sql/executor.go

```diff
@@ -192,14 +190,38 @@ func (e *Executor) execStmt(stmt parser.Statement, params parameters, planMaker
 			return err
 		}
 
-		result.Columns = plan.Columns()
-		for plan.Next() {
-			values := plan.Values()
-			row := driver.Result_Row{Values: make([]driver.Datum, 0, len(values))}
-			for _, val := range values {
-				if val == parser.DNull {
-					row.Values = append(row.Values, driver.Datum{})
-				} else {
+		switch stmt.StatementType() {
+		case parser.Ack:
+			// Send back an empty response.
+			result.Union = &driver.Response_Result_Rows_{
```

Comment:
The `Ack` statements are typically called with `sql.DB.Exec`, not `sql.DB.Query` (and equivalent interfaces in other languages), so it might be more appropriate to return a `RowsAffected` with a count of 0 instead of an empty set of rows (or have an explicit result marker type like we have for DDL).